management-service/services/mlp_service.go

```diff
@@ -97,7 +97,7 @@ func (service mlpService) refreshProjects() error {
 	ctx, cancel := context.WithTimeout(context.Background(), mlpQueryTimeoutSeconds*time.Second)
 	defer cancel()
 
-	projects, resp, err := service.mlpClient.api.ProjectApi.ProjectsGet(ctx, nil)
+	projects, resp, err := service.mlpClient.api.ProjectApi.V1ProjectsGet(ctx, nil)
```

Comment:
This is a consequence of removing `v1/` from the base path in: https://github.com/caraml-dev/mlp/pull/85

The change was applied to Turing / Merlin already. Eg: https://github.com/caraml-dev/turing/pull/338